api/v1beta2/bfdprofile_types.go

```diff
@@ -0,0 +1,72 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// BFDProfileSpec defines the desired state of BFDProfile.
+type BFDProfileSpec struct {
+	// +kubebuilder:validation:Maximum:=60000
+	// +kubebuilder:validation:Minimum:=10
+	ReceiveInterval *uint32 `json:"receiveInterval,omitempty"`
+	// +kubebuilder:validation:Maximum:=60000
+	// +kubebuilder:validation:Minimum:=10
+	TransmitInterval *uint32 `json:"transmitInterval,omitempty"`
+	// +kubebuilder:validation:Maximum:=255
+	// +kubebuilder:validation:Minimum:=2
+	DetectMultiplier *uint32 `json:"detectMultiplier,omitempty"`
+	// +kubebuilder:validation:Maximum:=60000
+	// +kubebuilder:validation:Minimum:=10
+	EchoInterval *uint32 `json:"echoInterval,omitempty"`
```

Comment:
What about echoTransmitInterval? I see it in the design proposal

Reply:
That was because the proposal was written when we were aiming at frr 8. Frr 7.5 only has echo interval.
That was the same in the operator, and in the configmap.